services/user-tenant-service/src/main/java/com/sourcefuse/jarc/services/usertenantservice/core/GenericEditor.java

```diff
@@ -0,0 +1,39 @@
+package com.sourcefuse.jarc.services.usertenantservice.core;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.beans.PropertyEditorSupport;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.commons.lang3.StringUtils;
+
+@Slf4j
+public class GenericEditor<T> extends PropertyEditorSupport {
+
+  private ObjectMapper objectMapper;
+  private Class<T> targetType;
+
+  public GenericEditor(ObjectMapper objectMapper, Class<T> targetType) {
+    this.objectMapper = objectMapper;
+    this.targetType = targetType;
+  }
+
+  public GenericEditor(ObjectMapper objectMapper) {
+    this.objectMapper = objectMapper;
+  }
+
+  @Override
+  public void setAsText(String text) throws IllegalArgumentException {
+    if (StringUtils.isEmpty(text)) {
+      setValue(null);
+    } else {
+      T obj;
+      try {
+        log.info("!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!" + text);
```

Comment:
change this log message with meaningful message

Reply:
done